example.py

```diff
@@ -9,19 +9,15 @@
 
 def uav_example():
     """A basic example of how to use the UAV agent."""
-    env = holodeck.make("InfiniteForest-MaxDistance")
-
-    # This changes the control scheme for the uav
-    env.agents["uav0"].set_control_scheme(ControlSchemes.UAV_ROLL_PITCH_YAW_RATE_ALT)
```

Comment:
Why remove this? Is this the default now? I think the idea was that it was showing that you can change this

Reply:
I figured it was redundant since the config files control that, but I think it is a good example and will put it back in as a comment